build/stylesheet/lib/extract-styles.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import * as emotion from 'react-emotion';
+import * as ReactTestRenderer from 'react-test-renderer';
+import { ThemeProvider } from 'emotion-theming';
+
+import { standard } from '../../../src/themes';
```

Comment:
Do we want to make the theme configurable?

Reply:
Yes. Makes total sense for when we start polishing this for release.